Resources/Textures/Clothing/Head/Hats/safarihat.rsi/meta.json

```diff
@@ -1,18 +1,26 @@
 {
-  "version": 1,
-  "license": "CC0-1.0",
-  "copyright": "Created by EmoGarbage404",
-  "size": {
-    "x": 32,
-    "y": 32
-  },
-  "states": [
-    {
-      "name": "icon"
+    "version": 1,
+    "license": "CC0-1.0",
+    "copyright": "Created by EmoGarbage404, Inhands by onesch",
+    "size": {
+      "x": 32,
+      "y": 32
     },
-    {
-      "name": "equipped-HELMET",
-      "directions": 4
-    }
-  ]
-}
+    "states": [
+      {
+        "name": "icon"
+      },
+      {
+        "name": "equipped-HELMET",
+        "directions": 4
+      },
+      {
+        "name": "inhand-left",
+        "directions": 4
+      },
+      {
+        "name": "inhand-right",
+        "directions": 4
+      }
+    ]
+  }
```

Comment:
Please keep the indentation as it was before at two spaces per level.
Also make sure the closing curly brackets line up correctly.
I'll ask an art director for approval for the sprite.